docs/content/tutorials/cluster.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: doc_page
+---
+
+# Clustering
+
+Druid is designed to be deployed as a scalable, fault-tolerant cluster.
+
+In this document, we'll set up a simple cluster and discuss how it can be further configured to meet
+your needs. This simple cluster will feature scalable, fault-tolerant servers for Historicals and MiddleManagers, and a single
+coordination server to host the Coordinator and Overlord processes. In production, we recommend deploying Coordinators and Overlords in a fault-tolerant
+configuration as well.
```

Comment:
as an user: what does "deploying Coordinators and Overlords in a fault-tolerant configuration" mean?